package.json

```diff
@@ -42,7 +42,7 @@
   },
   "peerDependencies": {
     "@segment/analytics-react-native": "*",
-    "@appcues/react-native": "^3.1.0"
+    "@appcues/react-native": "^3.1.1"
```

Comment:
ignoring the example app updates - this is the actual version bump for the plugin